opae.spec.in

```diff
@@ -272,6 +275,7 @@ ldconfig
 @CMAKE_INSTALL_PREFIX@/bin/afu_sim_setup
 @CMAKE_INSTALL_PREFIX@/bin/with_ase
 @CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/libase*
+@CMAKE_INSTALL_PREFIX@/@OPAE_LIB_INSTALL_DIR@/libopae-c-ase*
```

Comment:
I think that as of [#116](https://github.com/OPAE/opae-libs/pull/116), the documentation will have to be updated to say that -DBUILD_ASE=yes is always given when the intention is to build RPMs.

Reply:
Ah yes.. I should have looked at the preceding lines.  Please note that BUILD_ASE should be deprecated in favor of OPAE_BUILD_SIM.  Thanks